### PR TITLE
i18n(zh-cn): update guides/configuring-astro.mdx & guides/data-fetching.mdx

### PR DESCRIPTION
**Pull request by zhuchentong**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- update zh-cn/guides/configuring-astro.mdx
- update zh-cn/guides/data-fetching.mdx

### CODE REVIEW

**Review by liruifengv — CHANGES_REQUESTED** [review]
本页还有很多和英文文档不一致的地方，可以对照英文文档仔细对比一下

**Comment by zhuchentong** [comment]
其他部分已经对照英文文档进行比对修改

File: src/content/docs/zh-cn/guides/configuring-astro.mdx [hunk 1]
```diff
@@ -20,18 +20,18 @@ export default defineConfig({
 })
 ```
 
-推荐使用 `defineConfig()` 以便 IDE 可以提供自动类型提示。一个最基本的有效配置文件应该是这样：
+推荐使用 `defineConfig()` 以便 IDE 可以提供自动类型提示，但这并非是必须的。一个最基本的有效配置文件应该是这样：
 
 ```js
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/configuring-astro.mdx:25]:
```suggestion
```js title="astro.config.mjs"
```

File: src/content/docs/zh-cn/guides/data-fetching.mdx [hunk 2]
```diff
@@ -8,7 +8,9 @@ description: 学习如何使用 Astro fetch API 获取远程数据。
 
 ## Astro 中的 `fetch()`
 
-所有 [Astro 组件](/zh-cn/core-concepts/astro-components/) 都可以在它们的组件脚本中使用[全局 `fetch()` 函数](https://developer.mozilla.org/zh-CN/docs/Web/API/fetch)来发起 API 请求。fetch 调用将会在构建时执行，并且数据都可用于组件模板中来生成动态 HTML。如果启用 SSR 模式，任何 fetch 调用都将在运行时执行。
+所有 [Astro 组件](/zh-cn/core-concepts/astro-components/) 都可以在它们的组件脚本中使用[全局 `fetch()` 函数](https://developer.mozilla.org/zh-CN/docs/Web/API/fetch)来使用完整的 URL 发起 API 请求。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/data-fetching.mdx:11]: 这里似乎有个举例：https://github.com/withastro/docs/commit/1f3c047522c85e3978c8b1538059e3345a1a0e7e

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Comment by liruifengv** [comment]
A little change. Other LGTM.

**Comment by zhuchentong** [comment]
@liruifengv Already modified as suggested

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @zhuchentong and @liruifengv 💜

**Pull request merged**